### PR TITLE
TokenFactory and SampleConsumer

### PR DESCRIPTION
**Pull request by sag-asya**
EPL code for refreshing the access tokens in memory without having to redeploy the EPL app.

### CODE REVIEW

**Comment by sag-tgo** [comment]
Thanks Ashish.
One general comment is to please format the code & comments so that the lines are not too long. 80 chars max would be good. This will help to avoid needing to scroll horizontally when this code is pasted into the blog post.

File: TokenFactory.mon [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+/** @private Internal event RegistrationCancellationReq */
+event RegistrationCancellationReq {
+	constant string SEND_CHANNEL := "TOKEN_FACTORY_IN_CHANNEL";
+	string eventType;
+}
+
+/** Use TokenRegistrar.registerInterest to get update callbacks for a particular token */
+event TokenRegistrar {
+	
+	/** @private */ listener responseListener;
```
Comment (sag-tgo) [inline TokenFactory.mon]: Formatting wise I think the comment and declaration should be on separate lines.

File: TokenFactory.mon [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	/** @private */ listener updateListener;
+	/** @private */ string eventType;
+	
+	/** A static method to be called by any consumers which are interested in getting refresh updates for a particular token
```
Comment (sag-tgo) [inline TokenFactory.mon]: I would prefer if all Apamadoc comments were formatted like this:
```
/**
 * My apamadoc comment
 * @returns something
 */
```

**Comment by sag-asya** [comment]
@sag-tgo, I have addressed your review comments. Please take a look. Lets try to conclude this asap

**Pull request merged**